.npmrc

```diff
@@ -0,0 +1,3 @@
+fund=false
+lockfile-version=3
```

Comment:
This is the key line: we pin to version 3

Reply:
Reviewing package-lock.json is tricky (Github can't cope with it... command-line git is fine)
You could note that:

line 3: `"lockfileVersion": 3,`
line 186:
```
    "node_modules/@antora/site-generator": {
      "version": "3.1.4",
```
so even though we've not, in the end, added site-generator explicitly, it is pulled in by the version 3 lockfile.

Reply:
For bonus points, check out this PR, and then

```
npm ci
npx antora -v
```
and check that both cli and site-generator are 3.1.4